bluemix/helpers.js

```diff
@@ -10,218 +10,225 @@ var g = require('strong-globalize')();
+        // is not present
+        when: !this.bluemix,
       };
 
-      prompts.push(appMemoryPrompt, appInstancesPrompt, appDomainPrompt,
-        appHostPrompt, appDiskQuotaPrompt);
+      var done = this.async();
+      this.prompt([manifestPrompt]).then(function(answers) {
+        this.enableManifest = answers.enableManifest || this.enableManifest;
+        var prompts = [];
+        if (this.enableManifest) {
+          var manifestFilePath = path.resolve('.', 'manifest.yml');
+          var manifest = {};
+          if (fs.existsSync(manifestFilePath)) {
+            manifest = yaml.load(fs.readFileSync(manifestFilePath));
+          }
+
+          var appMemoryPrompt = {
+            name: 'appMemory',
+            message: g.f('How much memory to allocate for the app?'),
+            default: manifest.memory || '256M',
+            filter: helpers.normalizeSize,
+            validate: helpers.validateAppMemory,
+            when: this.enableManifest,
+          };
+
+          var appInstancesPrompt = {
+            name: 'appInstances',
+            message: g.f('How many instances of app to run?'),
+            default: manifest.instances || 1,
+            validate: helpers.validateAppInstances,
+            when: this.enableManifest,
+          };
+
+          var appDomainPrompt = {
+            name: 'appDomain',
+            message: g.f('What is the domain name of the app?'),
+            default: manifest.domain || 'mybluemix.net',
+            validate: helpers.validateAppDomain,
+            when: this.enableManifest,
+          };
+
+          var appHostPrompt = {
+            name: 'appHost',
+            message: g.f('What is the subdomain of the app?'),
+            default: manifest.host || this.appName,
+            validate: helpers.validateAppHost,
+            when: this.enableManifest,
+          };
+
+          var appDiskQuotaPrompt = {
+            name: 'appDiskQuota',
+            message: g.f('How much disk space to allocate for the app?'),
+            default: manifest.disk_quota || '1G',
+            filter: helpers.normalizeSize,
+            validate: helpers.validateAppDiskQuota,
+            when: this.enableManifest,
+          };
+
+          prompts.push(appMemoryPrompt, appInstancesPrompt, appDomainPrompt,
+            appHostPrompt, appDiskQuotaPrompt);
+        }
+
+        var toolchainPrompt = {
+          name: 'enableToolchain',
+          message: g.f('Do you want to create toolchain files?'),
+          type: 'confirm',
+          default: true,
+          when: !this.bluemix,
+        };
+
+        var dockerPrompt = {
+          name: 'enableDocker',
+          message: g.f('Do you want to create Dockerfile?'),
+          type: 'confirm',
+          default: true,
+          when: !this.bluemix,
+        };
+
+        prompts.push(toolchainPrompt, dockerPrompt);
+
+        this.prompt(prompts).then(function(answers) {
```

Comment:
A nitpick: can you rewrite this using an arrow function and remove `.bind(this)`?

```ts
return this.prompt(prompts).then(answers => {
  // ...
});
```